downstream/modules/platform/con-gw-understanding-authenticator-mapping.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="gw-understanding-authenticator-mapping"]
+
+= Understanding authenticator mapping
+
+Authentication:: validates a user’s identity, typically through a username and password or a trust system.
+
+Authorization:: determines what an authenticated user can do once they are authenticated. 
```

Comment:
Just a nit: 
```suggestion
Authorization:: determines what an authenticated user can do after they are authenticated. 
```
I'll leave this one up to you. Here's the IBM guideline applicable [here](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o): 
"(Use with caution) once adverb
Use only to mean one time. Do not use as a conjunction to mean after or when."

Reply:
Leaving the "Once" phrase because the meaning is that after you enter the information one time, it becomes available in the drop-down going forward.